src/tests/navBar.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+// eslint-disable-next-line
+import NavBar from '../components/Navbar';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  const root = createRoot(div);
+  root.render(<NavBar />);
+});
```

Comment:
- [ ] Great job implementing required changes from previous reviewers. However, there's one failing test. I think the proper import should be :point_down: 
```
 import NavBar from '../components/navBar'

```
Kindly look into this and fix.

![test-error](https://user-images.githubusercontent.com/26597432/236638552-b6a441cf-86b7-4d32-a0a0-d96793924f78.png)